challenges/qr_code_component/site.webmanifest

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Unnecessary

Reply:
I have since deleted the file, it may have been one of the many changes I made when I was having an issue running the tests. Looked for several solutions some which suggested changing files. Only downgrading my node version to 16.14.0 when running tests offered the solution.